packages/react/src/components/PageHeader/PageHeader.tsx

```diff
@@ -19,6 +19,7 @@ import { useMatchMedia } from '../../internal/useMatchMedia';
 import { Text } from '../Text';
 import { DefinitionTooltip } from '../Tooltip';
 import { AspectRatio } from '../AspectRatio';
+import { Tabs } from '../Tabs/Tabs';
```

Comment:
I think we can avoid the naming conflict with an alias
```suggestion
import { Tabs as BaseTabs } from '../Tabs/Tabs';
```